lib/asset_hosts.rb

```diff
@@ -0,0 +1,36 @@
+class AssetHosts
+  def call(_source, request)
+    return Figaro.env.domain_name if asset_host.blank?
```

Comment:
If `ASSET_HOST` is required by Figaro, is it really possible for `asset_host.blank?` to be `true`? Or does `require_keys` allow a blank value?

Reply:
Figaro only checks for the presence of the key. If the key is present but the value is set to an empty string (I've seen this happen on other apps I've worked on), it won't raise an error. I will change this to `.empty?` This is probably an edge case, but I thought it wouldn't hurt to be extra safe.